app/controllers/application_controller.rb

```diff
@@ -76,7 +76,7 @@ def update_last_seen_at
 
   def update_contest_checkin
     if user_signed_in?
-      original_user.contest_relations.active.absent.each do |relation|
+      original_user.contest_relations.active.merge(ContestRelation.absent).each do |relation|
```

Comment:
Just checking, is this intentional? It still seems to work without this change.

Reply:
It was intentional (rails 4 doesn't support chained scopes, though support is added in a later rails version) - so I replaced the ".active.absent" with this, but yeah that's interesting. I suspect it works without this because it just updates those relations that are already checked in to be checked_in again? given `ContentRelation#absent` is defined:

```
scope :absent, -> { where(checked_in: false) }
```

Reply:
It seems to be supported on Rails 4.2 at least: https://guides.rubyonrails.org/v4.2/active_record_querying.html#merging-of-scopes

Both versions perform the same query for me when I run them in rails console.

Reply:
right you are - i'm sure i tried this though.. either way, fixed!